other/edge-server/client/README.md

```diff
@@ -156,147 +152,38 @@ not contain information about running the Edge Server template. You can view
-
-You will need to provide the App Sevices App ID.
-If you don't know your App ID, you can get it from the CLI using:
-
-```shell
-appservices apps list
-```
-
-For other ways to find your App Services App ID, refer to
-[Find Your App ID](https://www.mongodb.com/docs/atlas/app-services/apps/metadata/#find-your-app-id)
-
-## Download, configure, and start the Edge Server
-
-To download, configure, and start the Edge Server, complete these steps:
-
-1. Get the Edge Server code.
-2. Complete the Edge Server configuration details.
-3. Install required dependencies to run the Edge Server.
-4. Run the Edge Server.
-
-This example application assumes that you're running the Edge Server and
-the example application on your local machine.
-
-### Get the Edge Server code
+## Set up Edge Server
 
-In the project's root directory, use wget to get the current Edge Server code
-as a .tar file:
-
-```shell
-wget --content-disposition https://services.cloud.mongodb.com/api/client/v2.0/tiered-sync/package/latest
-```
-
-Unzip the .tar to get the files:
-
-```shell
-tar -xvf *.tgz
-```
-
-You now have an `edge_server` directory containing the server files. For the
-project's npm scripts to work, the `edge_server` directory should be at the same
-level as the `node-server` and `react-client` directories:
-
-```shell
-| - <edge-server-template-app>
-|  | - node-server
-|  | - react-client
-|  | - edge_server
-```
-
-### Configure the Edge Server
-
-The `edge_server` directory contains a `config.json` file you must edit with
-the appropriate values to configure the server.
-
-Replace the `clientAppId` value with the App Services App ID for the Edge Server-enabled App
-you created above. For information about how to find the App Sevices App ID,
-refer to
-[Find Your App ID](https://www.mongodb.com/docs/atlas/app-services/apps/metadata/#find-your-app-id).
-
-Replace the `cloudSyncServerAuthSecret` value with the auth secret for your
+Before you can run the template app in this repo, you need to have a functional
 Edge Server.
 
-For more details about this configuration file, refer to the
-[Configure Edge Server](https://www.mongodb.com/docs/atlas/app-services/edge-server/configure/#complete-the-edge-server-configuration-details)
-documentation.
-
-### Install required dependencies to run the Edge Server
-
-The `edge_server` directory contains a `README.md` file with instructions
-for installing dependencies and starting the Edge Server. The host running
-the Edge Server must have `docker`, `docker-compose`, `make` and `jq`
-installed.
-
-For computers running macOS, we recommend using `brew` to install required
-dependencies.
-
-### Run the Edge Server
+To set up an Edge Server:
 
-Make sure Docker is running on your local machine. Then, run the following
-command to start the Edge Server:
+1. [Create a Service for Edge Server](https://www.mongodb.com/docs/atlas/app-services/edge-server/manage-edge-servers/#create-a-service-for-edge-server)
```

Comment:
Again, we can skip the "Create a Service" step and change the numbering on the subsequent steps if the user has created a template app.